test/run-tests

```diff
@@ -33,6 +34,35 @@ def sighandler_exit(signo, frame):
     sys.exit(0)
 
 
+def handle_httperror(error):
+    """
+    Try to handle HTTPError.
+    Tries to handle transient errors by waiting.
+
+    Return True if error is handled.
+    """
+
+    # Handle transient server-side errors
+    if error.response.status_code in (500, 502, 503, 504):
+        print(
+            f">>> Server returned {error.response.status_code} {error.response.reason}, waiting..."
+        )
+        time.sleep(600)
```

Comment:
Good idea, though I'm not sure about best way to do it

Reply:
Since it not a method of an object, I believe the best way is to use a global variable, for example `transient_error_wait_time`, use it with `global transient_error_wait_time` in `handle_transient_httperrors()` and reset it to a default value when no exception occured for `handle_transient_httperrrors()` to handle. The values should be constants.

Eventually, the code should probably be changed to also retry at least the requests to report the status back but I am not sure if they fail often and it is worth the effort now.